services/order_by.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+from django.utils.translation.trans_real import translation
```

Comment:
The import `from django.utils.translation.trans_real import translation` is incorrect for atomic transactions. You should use `from django.db import transaction` instead.